packages/ui-components/Common/CircularIcon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+
+import styles from './index.module.css';
+
+interface CircularIconProps {
+  symbol: string;
+  color: string;
```

Comment:
What if we have something like kind: "sting" | "object" ....

Reply:
I think we should not pass the color code as props, I am in favor of adding the possibilities here as kind and defining the color set here. For example, you can look at the badge component;

https://github.com/nodejs/nodejs.org/blob/main/packages/ui-components/Common/Badge/index.tsx#L6

https://github.com/nodejs/nodejs.org/blob/main/packages/ui-components/Common/Badge/index.module.css#L21-L37

This allows us to choose a background with the right contrast ratio with the background colors and not go beyond our color system